pkg/httputil/httputil.go

```diff
@@ -69,7 +69,10 @@ func ErrorEncoder(_ context.Context, err error, w http.ResponseWriter) {
 	}
 	w.WriteHeader(code)
 
-	enc.Encode(errMap)
+	err = enc.Encode(errMap)
+	if err != nil {
+		fmt.Println(errors.Wrap(err, "encoding errMap"))
```

Comment:
need to either use the logger or write a 500. I think not checking the error here is preferable. Failing to encode just means we can't write back to the client for some reason.